merlin/models/tf/inputs/base.py

```diff
@@ -218,6 +218,7 @@ def InputBlockV2(
     pre: Optional[BlockType] = None,
     post: Optional[BlockType] = None,
     aggregation: Optional[TabularAggregationType] = "concat",
+    **kwargs,
```

Comment:
if we're passing these kwargs to the `Embeddings`, maybe can rename this to `embeddings_kwargs`? 

And why would these `kwargs` on the input block be passed to the embeddings instead of the ParallelBlock for example? If the embeddings needs to be customized from the default that can already be achieved by passing in the `embeddings` param?

Reply:
Makes sense @oliverholworthy . As the user can always provide the `embeddings` as an argment, the **kwargs can be forwarded to another block (`ParallelBlock`)